src/lib/final_report.ml

```diff
@@ -3,6 +3,11 @@ open Nonstd
 module String = Sosa.Native_string
 let (//) = Filename.concat
 
+
+(** Makes the below equivalent.
+let x = canonicalize "/dsde/deds///desd//de/des/"
+let y = canonicalize "./dsde/deds///desd//de/des/"
+let z = canonicalize "../../dsde/deds///desd//de/des/" *)
```

Comment:
Since you use `(** ..*)` you could put it a code-bloc thing maybe? 
(http://caml.inria.fr/pub/docs/manual-ocaml/ocamldoc.html#sec333 → `{[ let ... ]}`)